contrib/inventory_builder/inventory.py

```diff
@@ -238,7 +238,7 @@ def ips(start_address, end_address):
             return [ip_address(ip).exploded for ip in range(start, end + 1)]
 
         for host in hosts:
-            if '-' in host and not host.startswith('-'):
+            if '-' in host and not (host.startswith('-') or host[0].isalpha()):
```

Comment:
Took me some time to figure out that if the first char is a letter, it has to be a host, not a range. This is okay